diagnostics/model_test/README.md

```diff
@@ -1,23 +1,25 @@
 # Compare model output between two different backends
 
-This model_test tool will run the model inference seperately on any two specified backends(e.g. Backends: Tensorflow (native), nGraph-CPU, nGraph-GPU etc) in json file and the outputs from each backend should match given the same inputs. It can be used as a debugging tool for layer by layer comparison, and also a verification that nGraph produces the same output as Tensorflow. 
+This model_test tool will run the model seperately on any two specified backends(e.g. Backends: Tensorflow (native), nGraph-CPU, nGraph-GPU etc) in json file and the outputs from each backend should match given the same inputs. It can be used as a debugging tool for layer by layer comparison, and also a verification that nGraph produces the same output as Tensorflow.  Depending on the input model specified in the .pb, .pbtxt or .ckpt it can be used to verify layers in forward pass and backward pass both.
 
 # Required files to use the tool:
-* A json file: Provide model specific parameters. Look at the example ```mnist_cnn.json```. You can start with the ```template.json``` and modify it to match your model
-* A tensorflow model file: Either .pb or .pbtxt file
+* A json file: Provide model specific parameters. Look at the example ```mnist_cnn.json``` to use a .pb or .pbtxt graph and ```mnist_mlp.json``` for checkpoint graph
+* For inference verification,a tensorflow model inference .pb or .pbtxt graph file is required
```

Comment:
```suggestion
* For inference verification, a tensorflow model inference .pb or .pbtxt graph file is required
```

Reply:
For inference verification, a tensorflow model inference .pb or .pbtxt graph file is required:::

Not sure about this statement. we can do inf on pb, pbtxt & ckpt, but training comparision only on ckpt.